src/view/domconverter.js

```diff
@@ -224,7 +224,12 @@ export default class DomConverter {
 				return domElement;
 			} else {
 				// Create DOM element.
-				domElement = domDocument.createElement( viewNode.name );
+				const namespace = viewNode.getAttribute( 'xmlns' );
```

Comment:
I think this would be enough:

```js
if ( viewNode.hasAttribute( 'xmlns' ) )
```

Reply:
How do you mean? I need the namespace for the constructor of the element.

Reply:
Ah understood. Will change.